docs/_includes/Modules.md

```diff
@@ -129,29 +135,42 @@ module Mod {
 Note that inside `m()`, we have to use `A` instead of `Helpers`, as we bound
 it to a different name. The name `Helpers` is not available inside `m()`,
 as only names that have been bound inside `Mod` are available. In order
-to use the members from another module, it either has to be declared
+to use the members from another module, that other module either has to be declared
 there with `module` or imported with `import`.
 
 We don't have to give `Helpers` a new name, though, if we don't want
-to. We can write `import Helpers = Helpers` if we want to, and Dafny
+to. We can write `import Helpers = Helpers` to import the module under
+its own name; Dafny
 even provides the shorthand `import Helpers` for this behavior. You
 can't bind two modules with the same name at the same time, so
 sometimes you have to use the = version to ensure the names do not
-clash.
+clash. When importing nested modules, `import B.C` means `import C = B.C`;
+the new name is always the last name segment of the module designation.
 
 The ``QualifiedModuleName`` in the ``ModuleImport_`` starts with a
 sibling module of the importing module, or with a submodule of the
-importing module. There is no wya to refer to the parent module, only
+importing module. There is no way to refer to the parent module, only
 sibling modules (and their submodules).
 
+Import statements may occur at the top-level of a file as well.
+There they serve simply as a way to givve a new name, perhaps a 
+shorthand name, to a module. For example,
+
+```
+module MyModule { ... } // declares module MyModule
+import MyModule  // error: cannot add a moduled named MyModule
+                 // because there already is one
+import M = MyModule // OK. M and MyModule are equivalent
+```
+
 ## Opening Modules
 
 Sometimes, prefixing the members of the module you imported with the
 name is tedious and ugly, even if you select a short name when
 importing it. In this case, you can import the module as `opened`,
 which causes all of its members to be available without adding the
-module name. The `opened` keyword must immediately follow `import`, if it
-is present. For example, we could write the previous example as:
+module name. The `opened` keyword, if present, must immediately follow `import`.
+For example, we could write the previous example as:
```

Comment:
It would be nice to say that the Dafny style guidelines suggest using opened modules only sparingly.

Reply:
Added

Reply:
Done